src/gui/guiFormSpecMenu.h

```diff
@@ -405,7 +405,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 	std::vector<std::pair<FieldSpec,gui::IGUICheckBox*> > m_checkboxes;
 	std::map<std::string, TooltipSpec> m_tooltips;
 	std::vector<std::pair<irr::core::rect<s32>, TooltipSpec>> m_tooltip_rects;
-	std::vector<std::pair<FieldSpec,gui::IGUIScrollBar*> > m_scrollbars;
+	std::vector<std::pair<FieldSpec,guiScrollBar*> > m_scrollbars;
```

Comment:
Does this even work without including `"guiScrollBar.h"` in `guiFormSpecMenu.h`?

Reply:
I agree, either the include should be moved to the header or use a forward declaration, not sure which is preferred in MT but since `guiTable.h` is included in the header, I would go with the former.